src/openid.js

```diff
@@ -38,44 +50,43 @@ const getTokens = (code, state, host) =>
     .then(slackToken => {
       logger.debug('Got token: %s', JSON.stringify(slackToken), {});
 
-      return getUserInfo(slackToken.access_token).then(userInfo => {
-        logger.debug('Got user details: %s', JSON.stringify(userInfo), {});
+      const userInfo = getUserInfoFromTokenResponse(slackToken);
```

Comment:
this avoids the additional user info request, since the identity data comes with the token response